.github/workflows/unmanaged-dependabot.yml

```diff
@@ -19,6 +22,9 @@ jobs:
         uses: actions/checkout@v4
       - name: Generate Dependabot Config
         uses: fredrikaverpil/dependabot-generate@v1
+        with:
+          interval: "monthly"
+          ignore-dirs: ".tools"
```

Comment:
Figure out a way to add custom entries, like for `.tools`, which should have `dependency-types: indirect`.